thonny/running.py

```diff
@@ -865,6 +865,64 @@ def open_custom_system_shell(self):
         raise NotImplementedError()
 
 
+class EnvFile(object):
+    def __init__(self, fnam=None):
+        self.fnam = fnam if fnam else ".env"
```

Comment:
Why not simply use `".env"` as the default value?

Reply:
you mean like ?
def __init__(self, fnam=".env"):

pep 671 status is "draft" 
https://www.python.org/dev/peps/pep-0671/

so i would propose to add an comment
# todo pep 671

Reply:
with a function encapsulating `env_file = os.environ.get("THONNY_ENV_FILE", ".env")` so that the strig ".env" does not appear at several places

Reply:
As far as I can see, on this line pep 671 is not relevant, as the default value is fixed and does not need to be late bound. Anyway, this is a minor issue, so feel free to keep it.

Reply:
changed the implemenation so that no default is used anymore here